tests/tests/gas_price.rs

```diff
@@ -590,7 +611,7 @@ fn produce_block__l1_committed_block_affects_gas_price() {
             // Won't accept DA costs until l2_height is > 1
             driver.client.produce_blocks(1, None).await.unwrap();
             // Wait for DaBlockCosts to be accepted
-            tokio::time::sleep(Duration::from_millis(2)).await;
+            tokio::time::sleep(Duration::from_millis(200)).await;
             // Produce new block to update gas price
             driver.client.produce_blocks(1, None).await.unwrap();
```

Comment:
@MitchTurner this test seems to be failing. was it passing locally for you?

Reply:
https://github.com/FuelLabs/fuel-core/actions/runs/12759398333 failing from over 6hr ago, perhaps a bad merge

Reply:
fixed now